tests/by-util/test_ls.rs

```diff
@@ -2099,6 +2099,35 @@ fn test_ls_order_time() {
     }
 }
 
+#[test]
+fn test_ls_order_mtime() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    at.touch("test-3");
+    at.append("test-3", "333");
+    sleep(Duration::from_millis(100));
```

Comment:
I think this is a use case for the [set_modified](https://doc.rust-lang.org/std/fs/struct.File.html#method.set_modified) function introduced with Rust 1.75. It would allow you to get rid of all the `sleep` calls.

Reply:
much better, thanks